pkg/handler/v2/external_cluster/external_cluster.go

```diff
@@ -624,6 +625,26 @@ func PatchEndpoint(userInfoGetter provider.UserInfoGetter, projectProvider provi
 	}
 }
 
+func CheckContainerRuntime(ctx context.Context,
```

Comment:
So I think this is bad architecture. The controllers are now calling code from the API handlers, which they should have no knowledge of whatsoever. Now the controllers cannot work without our API and since we might move the API out of this repo in the mid future, this will break.

Can you restructure your code to put commonly used code outside of the API handlers (i.e. outside of `pkg/handler/`)?

Reply:
Good Point!
I am moving this method to pkg/kubernetes